x-pack/legacy/plugins/xpack_main/server/lib/__tests__/setup_xpack_main.js

```diff
@@ -39,7 +41,7 @@ describe('setupXPackMain()', () => {
         elasticsearch: mockElasticsearchPlugin,
         xpack_main: mockXPackMainPlugin
       },
-      newPlatform: { setup: { plugins: { features: {} } } },
+      newPlatform: { setup: { plugins: { features: {}, licensing: { license$: new BehaviorSubject() } } } },
```

Comment:
1. I will add a mock for NP the plugin.
2. This file uses Sinon instead of Jest. I don't want to provide the full mock because we are about to delete this setup completely.